packages/flutter_tools/lib/src/version.dart

```diff
@@ -159,7 +161,7 @@ class FlutterVersion {
     final String flutterText = 'Flutter$versionText • channel $channel • ${repositoryUrl ?? 'unknown source'}';
     final String frameworkText = 'Framework • revision $frameworkRevisionShort ($frameworkAge) • $frameworkCommitDate';
     final String engineText = 'Engine • revision $engineRevisionShort';
-    final String toolsText = 'Tools • Dart $dartSdkVersion';
+    final String toolsText = 'Tools • Dart $dartSdkVersion • DevTools $devToolsVersion';
```

Comment:
This LGTM. We should also add this to the doctor (which is unfortunately a different codepath): https://github.com/christopherfujino/flutter/blob/f598cb2f35c1de110f2a809429dbfb41bc8d8e84/packages/flutter_tools/lib/src/doctor.dart#L433

Reply:
done - ptal